From 6ea7a634070bb7a0312fcd6abd00b45cf7dd46f1 Mon Sep 17 00:00:00 2001
From: Michael Forney <mforney@mforney.org>
Date: Sun, 21 Apr 2019 13:43:27 -0700
Subject: [PATCH] Restore NPred to its previous value

This now only limits the number of arguments when parsing the input SSA,
which is usually a small fixed size (depending on the frontend).
---
 all.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/all.h b/all.h
index fba93b1..5f67e81 100644
--- a/all.h
+++ b/all.h
@@ -32,7 +32,7 @@ typedef struct Target Target;
 
 enum {
 	NString = 64,
-	NPred   = 127,
+	NPred   = 63,
 	NIns    = 1 << 20,
 	NAlign  = 3,
 	NField  = 32,
-- 
2.21.0

